listener/inventory_processor.py

```diff
@@ -221,12 +226,18 @@ async def _db_import_system_repos(self, conn, repo_list: list, system_id: int) -
                     NEW_SYSTEM_REPO.inc(len(repo_ids))
         return repo_ids
 
-    async def _db_unlink_system_repos(self, conn, repo_list: list, system_id: int) -> [int]:
+    async def _db_unlink_system_repos(self, conn, repo_list: list, system_id: int, repo_id_mapping: dict = None) -> [int]:
```

Comment:
no need for `None` default value in `repo_id_mapping: dict = None`, it's never called without the param